lib/mail.rb

```diff
@@ -9,26 +9,11 @@ module Mail # :doc:
   require 'net/smtp'
   require 'mini_mime'
 
-  if RUBY_VERSION <= '1.8.6'
-    begin
-      require 'tlsmail'
-    rescue LoadError
-      raise "You need to install tlsmail if you are using ruby <= 1.8.6"
-    end
-  end
-
-  if RUBY_VERSION >= "1.9.0"
-    require 'mail/version_specific/ruby_1_9'
-    RubyVer = Ruby19
-  else
-    require 'mail/version_specific/ruby_1_8'
-    RubyVer = Ruby18
-  end
+  require 'mail/version_specific/ruby_1_9'
```

Comment:
@deivid-rodriguez is this needed any more if we are not supporting 1.9?

Reply:
I think the module is still needed, but the `ruby_1_9` naming is now misleading since there's now a single implementation of these methods. I guess it could be renamed to something different, for example, `Mail::Utils`.

Reply:
Please do implement.

Reply:
There's a `Mail::Utilities` module already. I think it makes sense to move these methods there.

Reply:
Alright, for now I removed the `Ruby19` module and the `ruby_1_9.rb` file in favor of the `RubyVer` module (which existed already), and a `version_specific.rb` file. Still not great since most of the stuff at `version_specific.rb` is not really version specific anymore. But seems much better than before and it was an easy change.

Reply:
I guess best would be to move everything to `Mail::Utilities`, I'll do it tomorrow first thing since now I have to leave the computer :)

Reply:
Done!